features/small.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <small>
-description: The `<small>` element TODO.
+description: The `<small>` element represents side-comments and small print, like copyright and legal text, independent of its styled presentation.
```

Comment:
Can you also say that it makes text smaller?

Reply:
```suggestion
description: The `<small>` element represents side-comments and small print, like copyright and legal text, independent of its styled presentation. Typically styled in a reduced font size by default.
```